pkg/oc/cli/describe/chaindescriber_test.go

```diff
@@ -1,19 +1,17 @@
 package describe
 
 import (
-	"strings"
 	"testing"
 
 	"github.com/gonum/graph"
 	"github.com/gonum/graph/concrete"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/util/sets"
-	kapi "k8s.io/kubernetes/pkg/api"
-
-	"github.com/openshift/origin/pkg/client/testclient"
-	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
 )
 
+/*
+* FIXME: Disable because we need a better way to feed the multiple fake
```

Comment:
This should compile.  Can you just rename it or skip it?

Reply:
Oh, it won't compile, will it.....

Reply:
@deads2k i will have to plumb the invocation of `NewChainDescriber` to take all clients that we need to make this compile so we can skip it, just don't see the point if we are going to rework the entire test anywayt.

Reply:
> @deads2k i will have to plumb the invocation of NewChainDescriber to take all clients that we need to make this compile so we can skip it, just don't see the point if we are going to rework the entire test anywayt.

I doubt we refactor the entire test.  More likely a filter.